docs/DeveloperGuide.md

```diff
@@ -423,14 +424,14 @@ Use case ends.
 8.  The app should have a log of user actions for debugging purposes.
 9.  The app should help users validate their inputs for their client's claim information as they are adding or deleting the inputs based on known market standards.
 
-*{More to be added}*
-
 ### Glossary
 
 * **Mainstream OS**: Windows, Linux, Unix, MacOS
 * **Insurance Agent**: The user of the app.
-* **Client**: A potential customer who is keen on purchasing an insurance plan or a customer who has purchased at least one insurance plan from the insurance agent.
-* **Insurance Plan ID**: A unique ID assigned to the insurance plan by the system.
+* **Client**: A customer who is keen on purchasing an insurance plan or a customer who has purchased at least one insurance plan from the insurance agent.
+* **Insurance Plan ID**: A unique ID assigned to the insurance plan by the system. 
+  * Supported insurance plans are: `Basic Insurance Plan` : `0` and `Travel Insurance Plan` : `1`.
+  * Allowing the user to add their own insurance plan types is a [planned enhancement](#planned-enhancements). 
```

Comment:
Should remove this as we are planning to not make it a planned enhancement.

Reply:
by this, i meant line 434

Reply:
Forgot to delete it after our discussion. Good catch!